runtime/nls/shrc/j9shr.nls

```diff
@@ -6929,3 +6929,10 @@ J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.explanation=Th
 J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.system_action=The JVM continues.
 J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.user_response=Contact your service representative.
 # END NON-TRANSLATABLE
+
+J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1=The page size of the operating system is different from the page size of the system on which this cache was created.
```

Comment:
The message for J9NLS_CC_OSPAGE_SIZE_MISMATCH can be removed since it's not used any more.